src/app/app.component.ts

```diff
@@ -163,4 +176,18 @@ export class AppComponent {
       switchMap(() => this.userSettings.userSetting$.pipe(take(1)))
     );
   }
+
+  private async initRouteNavigationLogger(): Promise<void> {
+    this.router.events.pipe(
+      filter((e): e is RouterEvent => e instanceof RouterEvent)
+    ).subscribe((event) => {
+      const url = removeOauthTokenFromUrl(event.url);
+      const type = event.constructor?.name;
```

Comment:
Hvis du vil slippe alt dette her så kan du bare kalle event.toString()? Feks `this.loggingService.debug(event.toString(), ROUTER_DEBUG_TAG)`. Hvis du gjør det så må du nok justere litt på removeOauthTokenFromUrl, endre til replaceAll siden urlen kan forekomme flere ganger.

Linje 189 mangler debug tag ser det ut til.

Reply:
God ide! Tenkte ikke på at jeg kunne bruke samme logikk for å fjerne token.

Reply:
Har gjort som du foreslo:)